jf_agent/jf_jira/jira_download.py

```diff
@@ -495,7 +495,10 @@ def download_necessary_issues(
                     new_issues_this_batch.append(issue)
             prog_bar.update(len(new_issues_this_batch))
 
-            yield _filter_changelogs(new_issues_this_batch, include_fields, exclude_fields)
+            if not include_fields and not exclude_fields:
+                yield new_issues_this_batch
+            else:
```

Comment:
I think this is unnecessary — if there’s no include or exclude the other block already handles the case properly